### PR TITLE
Use Ground logo for fav icon

### LINKED ISSUE #714: Use Ground logo for favicon instead of Angular icon
@parulraheja98 Could you PTAL?

Steps should be something like this:
https://dev.to/prachi/simple-steps-to-replace-your-favicon-icon-in-angular-277b

### PR DESCRIPTION
**Pull request by semla**
Fixes #714

### CODE REVIEW

File: web/angular.json [hunk 1]
```diff
@@ -19,7 +19,15 @@
             "polyfills": "src/polyfills.ts",
             "tsConfig": "tsconfig.app.json",
             "aot": true,
-            "assets": ["src/favicon.ico", "src/assets"],
+            "assets": [
```
Comment (gino-m) [inline web/angular.json:22]: Tests are failing with:

```
> ng test --browsers ChromeHeadlessNoSandbox --watch=false --code-coverage

Project 'web' contains extension with invalid name (lint).
Project 'web' contains extension with invalid name (e2e).
Project 'web' contains extension with invalid name (deploy).
Error: Schema validation failed with the following errors:
  Data path "/assets" must be array.
```

Any ideas?

**Review by gino-m — CHANGES_REQUESTED** [review]
Thanks for the PR! One Q inline.

File: web/angular.json (hunk 1 above)
Comment (semla) [reply web/angular.json:22]: Typos, I pushed a new commit.
Ok?

**Review by gino-m — APPROVED** [review]
LGTM, thanks!

**Pull request merged**